src/constants/permissions.ts

```diff
@@ -109,8 +109,14 @@ const values = {
     SYSCONSOLE_WRITE_PLUGINS: 'sysconsole_write_plugins',
     SYSCONSOLE_READ_INTEGRATIONS: 'sysconsole_read_integrations',
     SYSCONSOLE_WRITE_INTEGRATIONS: 'sysconsole_write_integrations',
-    SYSCONSOLE_READ_COMPLIANCE: 'sysconsole_read_compliance',
-    SYSCONSOLE_WRITE_COMPLIANCE: 'sysconsole_write_compliance',
+    SYSCONSOLE_READ_COMPLIANCE_DATA_RETENTION: 'sysconsole_read_compliance_data_retention',
+    SYSCONSOLE_WRITE_COMPLIANCE_DATA_RETENTION: 'sysconsole_write_compliance_data_retention',
```

Comment:
Nit:
Add `_policy` to the end here and update permission in backend accordingly.
```suggestion
    SYSCONSOLE_READ_COMPLIANCE_DATA_RETENTION_POLICY: 'sysconsole_read_compliance_data_retention_policy',
    SYSCONSOLE_WRITE_COMPLIANCE_DATA_RETENTION_POLICY: 'sysconsole_write_compliance_data_retention_policy',
```